pystiche_papers/sanakoyeu_et_al_2018/_loss.py

```diff
@@ -178,7 +179,6 @@ def transformer_loss(
     impl_params: bool = True,
     style_aware_content_kwargs: Optional[Dict[str, Any]] = None,
     transformed_image_kwargs: Optional[Dict[str, Any]] = None,
-    style_loss_kwargs: Optional[Dict[str, Any]] = None,
```

Comment:
Why did you remove this?

Reply:
I have removed this one because the `prediction_loss`  is not to be changed anymore. The `MultiPredictionOperator` (`prediction_loss`) should be the same for the `transformer_loss` and `discriminator_loss`.